app/views/layouts/application.html.erb

```diff
@@ -20,5 +19,6 @@
   <body>
     <%= render partial: 'layouts/flash' %>
     <%= yield %>
+    <%= javascript_pack_tag 'application' %>
```

Comment:
We don’t want this in the head?

Reply:
Good point, it should stay in the head (this was needed by the example I removed).